tests/e2e/specs/product-editor/classic-integration.test.js

```diff
@@ -0,0 +1,707 @@
+/**
+ * External dependencies
+ */
+import { expect, test, Page } from '@playwright/test';
+
+/**
+ * Internal dependencies
+ */
+import * as api from '../../utils/api';
+import { getClassicProductEditorUtils } from '../../utils/product-editor';
+
+test.use( { storageState: process.env.ADMINSTATE } );
+test.describe.configure( { mode: 'serial' } );
+
+test.describe( 'Classic Product Editor integration', () => {
+	/**
+	 * @type {Page}
+	 */
+	let page = null;
+	let editorUtils = null;
+
+	async function getAvailableProductAttributesWithTestValues( locator ) {
+		const {
+			gtin,
+			mpn,
+			brand,
+			condition,
+			gender,
+			size,
+			sizeSystem,
+			sizeType,
+			color,
+			material,
+			pattern,
+			ageGroup,
+			multipack,
+			isBundle,
+			availabilityDate,
+			availabilityTime,
+			adultContent,
+		} = editorUtils.getAllProductAttributes( locator );
+
+		const allPairs = [
```

Comment:
This is duplicated with block-integration equivalent. Can we refactor it?

Reply:
Adjusted in https://github.com/woocommerce/google-listings-and-ads/pull/2363/commits/f21dd78f0c3f994dcb1537a23b84c74fd69c4c09.